src/OpenTelemetry.Exporter.Geneva/Internal/Transports/UnixDomainSocketDataTransport.cs

```diff
@@ -73,7 +73,7 @@ private bool Connect()
         }
         catch (Exception ex)
         {
-            ExporterEventSource.Log.ExporterException("UDS Connect failed.", ex);
+            ExporterEventSource.Log.TransportException(nameof(UnixDomainSocketDataTransport), "Connection failure", ex);
```

Comment:
nit: Include something like "will retry connecting in background", to indicate this may be recovered.

Reply:
Updated